app/models/manageiq/providers/ansible_tower/configuration_manager/refresher.rb

```diff
@@ -0,0 +1,27 @@
+module ManageIQ::Providers
+  module AnsibleTower
+    class ConfigurationManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
+      include ::EmsRefresh::Refreshers::EmsRefresherMixin
+
+      def parse_inventory(manager, _targets)
+        manager.with_provider_connection do |connection|
+          manager.api_version = connection.version
+          manager.save
```

Comment:
We should populate that field for every ExtManagementSystem.

RHEV: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/redhat/infra_manager/refresher.rb#L12-L13